### DISCUSSION TITLE
Continue the question asked on Jul 18 in Q&A,I‘ve changed the governing equations in synxflow\apps\cudaLandslideSolverPybind\cuda_landslide_solver.cu,and When I run the following code "landslide.run(case_folder)", used to control the equations when modifying the equations in the file .Does the 'setup.py' can realize it?

### DISCUSSION THREAD

**Discussion by IlllIIIlllIl** [SynxFlow/SynxFlow#20] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/ec5539e6-eed3-45a3-a4c7-4954dd81ee1c)
![image](https://github.com/user-attachments/assets/b6fb88ba-0850-4c20-8c4c-8d963274f2f8)
the second figure is the file which I've changed the governing equation.

**Comment by xiaxilin** [comment, marked as answer]
Yes, this will recompile the code, you may want to uninstall the package before doing so.
